addons/l10n_id_efaktur_coretax/models/account_move_line.py

```diff
@@ -17,7 +17,7 @@ def _l10n_id_coretax_build_invoice_line_vals(self, vals):
         product = self.product_id
 
         # Separate tax into the regular and luxury component
-        luxury_tax = self.tax_ids.filtered(lambda tax: tax.tax_group_id == self.env.ref('l10n_id.l10n_id_tax_group_luxury_goods'))
+        luxury_tax = self.tax_ids.filtered(lambda tax: tax.tax_group_id == self.env.ref('l10n_id.l10n_id_tax_group_luxury_goods', raise_if_not_found=False))
```

Comment:
Please store the result of the ref and do an `if luxury_tax_group:`

Reply:
Also for the commit, please add the type of ticket/... this is for.

(In this case, opw-4635356)